keyboards/bandominedoni/bandominedoni/config.h

```diff
@@ -0,0 +1,368 @@
+
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+#define SOFT_SERIAL_PIN D2  // or D1, D2, D3, E6
+
+//#define LED_NUM_LOCK_PIN B0
+//#define LED_CAPS_LOCK_PIN B1
+//#define LED_SCROLL_LOCK_PIN B2
+//#define LED_COMPOSE_PIN B3
+//#define LED_KANA_PIN B4
+
+//#define BACKLIGHT_PIN B7
+//#define BACKLIGHT_LEVELS 3
+//#define BACKLIGHT_BREATHING
+
+#define RGB_DI_PIN D3
+//#ifdef RGB_DI_PIN
+//#    define RGBLED_NUM 16
+//#    define RGBLIGHT_HUE_STEP 8
+//#    define RGBLIGHT_SAT_STEP 8
+//#    define RGBLIGHT_VAL_STEP 8
+//#    define RGBLIGHT_LIMIT_VAL 255 /* The maximum brightness level */
+//#    define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+/*== all animations enable ==*/
+//#    define RGBLIGHT_ANIMATIONS
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
+/*== customize breathing effect ==*/
+/*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//#    define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+/*==== use exp() and sin() ====*/
+//#    define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//#    define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+//#endif
+
+#ifdef RGB_MATRIX_ENABLE
+
+/* ws2812 RGB MATRIX */
+#   define DRIVER_LED_TOTAL 76
+
+ // reacts to keypresses
+#   define RGB_MATRIX_KEYPRESSES
+
+//  for all fingers used at once.
+#   define LED_HITS_TO_REMEMBER 10
+
+#   define RGB_MATRIX_MAXIMUM_BRIGHTNESS 50
+#   define RGB_MATRIX_STARTUP_SPD 127
+// the above brighness setting has no effect on rgb_matrix_set_color().
+// Use darker colors instead.
+/*              RGB darker COLORS             */
+#   define RGB_DARKWHITE 0x33, 0x33, 0x33
+#   define RGB_DARKRED 0x33, 0x0, 0x0
+#   define RGB_DARKCORAL 0x33, 0x18, 0xF
+#   define RGB_DARKORANGE 0x33, 0x19, 0x0
+#   define RGB_DARKGOLDENROD 0x2B, 0x21, 0x6
+#   define RGB_DARKGOLD 0x33, 0x2B, 0x0
+#   define RGB_DARKYELLOW 0x33, 0x33, 0x0
+#   define RGB_DARKCHARTREUSE 0x19, 0x33, 0x0
+#   define RGB_DARKGREEN 0x0, 0x33, 0x0
+#   define RGB_DARKSPRINGGREEN 0x0, 0x33, 0x19
+#   define RGB_DARKTURQUOISE 0xE, 0x16, 0x15
+#   define RGB_DARKTEAL 0x0, 0x19, 0x19
+#   define RGB_DARKCYAN 0x0, 0x33, 0x33
+#   define RGB_DARKAZURE 0x1E, 0x31, 0x33
+#   define RGB_DARKBLUE 0x0, 0x0, 0x33
+#   define RGB_DARKPURPLE 0x18, 0x0, 0x33
+#   define RGB_DARKMAGENTA 0x33, 0x0, 0x33
+#   define RGB_DARKPINK 0x33, 0x19, 0x26
+
+//  https://docs.qmk.fm/#/feature_rgb_matrix
+//  Enable suspend mode.
+#   define RGB_DISABLE_WHEN_USB_SUSPENDED true
```

Comment:
```suggestion
#   define RGB_DISABLE_WHEN_USB_SUSPENDED
```

Reply:
Oh, that's why it didn't work. I'll remove the " true" as you suggested.

Reply:
Uh oh...
I got this error if I delete " true" here:
```
qmk_firmware $ make bandominedoni:led
QMK Firmware 0.13.2
Making bandominedoni with keymap led

avr-gcc (Homebrew AVR GCC 8.4.0_2) 8.4.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text	   data	    bss	    dec	    hex	filename
      0	  26786	      0	  26786	   68a2	.build/bandominedoni_led.hex

Compiling: quantum/rgb_matrix.c                                                                    quantum/rgb_matrix.c:70:36: error: operator '==' has no left operand
 #if RGB_DISABLE_WHEN_USB_SUSPENDED == false
                                    ^~
 [ERRORS]
 | 
 | 
 | 
make[1]: *** [.build/obj_bandominedoni_led/quantum/rgb_matrix.o] Error 1
make: *** [bandominedoni:led] Error 1

```
I pulled all the latest updates. Shall I keep the " true" here?

Reply:
Ugh, sorry, that's my fault, for now, leave it at "true"

Reply:
leave it at `true`, for now

Reply:
OK, it is left at `true`, at the momemt.